### PR TITLE
Unused Asset Nerfs

### PR DESCRIPTION
**Pull request by Cythisiax**
Unused Assets Nerfed. Possible future re-implementation when done right.

## About The Pull Request

Unused asset nerfs if properly done right. Possible re-implementation of future things. Brotherhood of Steel T-51B should be the BASELINE in terms of "top gear", and all items nerfed to T-51 status.

## Why It's Good For The Game

Nothing is implemented into the round, so it's not really "good for the game" As of yet.

## Changelog
Nerfed APA, X01 to T-51B(Green) stats. APA helmet, X01, and Hellfire nerfed to T-51B(Standard BoS issue stats. But funny linearmor shenanigans rework confusion.)

Nerfed Plasma Carbine, and Plasma Scatterer's fire delay +3, increased energy e_cost from 180-200. 12 shots via MFC. A few other things done as well.

Basically admin shit.

Will be implementing a proper "bunker" in the future with job slot spawns as well. Brotherhood zlevel implementation almost done. (Save underground space) zlevel will be on 3.

### CODE REVIEW

**Comment by Cythisiax** [comment]
Will be adjusting everything else when/if Adam's BoS rework PR is merged to better fit.
